### PR TITLE
[docs][windows] Improve CMake version details

### PR DESCRIPTION
**Pull request by enen92**
## Description
Found while setting up a build environment on windows, cmake version requirements are wrong for VS 2022.

### CODE REVIEW

**Review by thexai — APPROVED** [review]

File: docs/README.Windows.md [hunk 1]
```diff
@@ -37,16 +37,16 @@ git clone -b Matrix https://github.com/xbmc/xbmc kodi
 ```
 
 Several different strategies are used to draw your attention to certain pieces of information. In order of how critical the information is, these items are marked as a note, tip, or warning. For example:
- 
-**NOTE:** Linux is user friendly... It's just very particular about who its friends are.  
-**TIP:** Algorithm is what developers call code they do not want to explain.  
+
+**NOTE:** Linux is user friendly... It's just very particular about who its friends are.
```
Comment (bugwelle) [inline docs/README.Windows.md:41]: Line-break removal on purpose? :-)

Comment (enen92) [reply docs/README.Windows.md:41]: Which line break? There was only one on line 40 and it's still there (without whitespaces)

Comment (bugwelle) [reply docs/README.Windows.md:41]: The trailing two whitespaces are interpreted as a newline in markdown ([GFM](https://github.github.com/gfm/#hard-line-breaks)). You removed it, hence `TIP` won't be on a separate line).

Comment (enen92) [reply docs/README.Windows.md]: Well, guess we're always learning :) 
Thanks

Comment (bugwelle) [reply docs/README.Windows.md]: Thanks for updating the PR. Always nice to have a look at Team Kodi's (and your) PRs. I'm always leaning as well. :-)

**Pull request merged**